src/routes/github/github-router.ts

```diff
@@ -7,27 +7,37 @@ import { GithubConfigurationRouter } from "routes/github/configuration/github-co
 import { returnOnValidationError } from "../api/api-utils";
 import { header } from "express-validator";
 import { WebhookReceiverPost } from "./webhook/webhook-receiver-post";
+import { GithubServerAppMiddleware } from "middleware/github-server-app-middleware";
 
-export const GithubRouter = Router();
+const UUID_REGEX = "[0-9a-fA-F]{8}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{12}";
+
+export const GithubRouter = Router({ mergeParams: true });
+const subRouter = Router({ mergeParams: true });
+GithubRouter.use(`/:uuid(${UUID_REGEX})?`, subRouter);
```

Comment:
better put the `/:uuid(regex)?` here instead of in the rooter since this logic is specific to the github router.

Reply:
personally, I'm not a fan of using multiple routers within a router. The argument you're making can be said about specifying `/github` outside of the github router

Reply:
I agree this can be interpreted in many way. I am closing this thread, since:
1. It doesn't have bugs.
2. It doesn't have performance/security/etc issue.
3. It doesn't harm code clarity ( in my view improve though)

Let me know if you wish to re-open it.